libcloud/compute/drivers/cloudstack.py

```diff
@@ -1348,7 +1348,7 @@ def list_locations(self):
 
         return locations
 
-    def list_nodes(self, project=None):
+    def list_nodes(self, project=None, location=None):
```

Comment:
It looks like we missed this while merging the original changes - project should really be prefixed with `ex_` since it's not part of the standard API.

In any case, I'm fine with this for now.